auto_show/statistics.py

```diff
@@ -0,0 +1,20 @@
+from django.db.models import Sum, Q
+
+from auto_show.models import AutoShow
+from deals.models import Deal
+from deals.serializers import DealSerializer
+
+
+def statistics(pk):
+    auto_show = AutoShow.objects.get(pk=pk)
+    queryset_for_statistics = Deal.objects.filter(Q(participants="Producer-AutoShow") & Q(auto_shows=auto_show.id))
```

Comment:
We shouldn't use magic strings, i.e. `Producer-AutoShow`
Replace it with the value of `Participants. PRODUCER_AUTOSHOW`

Reply:
done